pystac_client/stac_api_io.py

```diff
@@ -218,6 +218,8 @@ def get_pages(
             Dict[str, Any] : JSON content from a single page
         """
         page = self.read_json(url, method=method, parameters=parameters)
+        if "features" in page and not page["features"]:
```

Comment:
`if not page.get("features")` handles both the case where there's no features property at all or it's an empty list

Reply:
Done (needed to be `if not page.get("features", True)`).